RELEASE-NOTES.md

```diff
@@ -1,5 +1,32 @@
 # Prefect Release Notes
 
+## Release 2.10.5
+
+### Deploy a Prefect flow via Github Actions
+
+With the new `Deploy a Prefect flow` Github action, you can deploy a flow to Prefect Cloud via the new [Projects](https://docs.prefect.io/latest/concepts/projects/) system.
+
+### Cloud Provider Workers
+
+Workers, Prefect's next-generation agents, have dedicated infrastructure types. This week, we are shipping types workers for each major cloud provider, AWS, GCP, and Azure. You will be able to find them in the [prefect-aws](https://github.com/PrefectHQ/prefect-aws), [prefect-gcp](https://prefecthq.github.io/prefect-gcp/), and [prefect-azure](https://github.com/PrefectHQ/prefect-azure) collections, respectively. 
+
+### Enhancements
+- In order to allow more space for the flow run graph, the metadata in right side bar of the flow run page has moved to the new "Details" tab - [#9258](https://github.com/PrefectHQ/prefect/pull/9258)
+
+### Fixes
+- fix(deployments): remove erroneous creation of work queue by @joelluijmes in https://github.com/PrefectHQ/prefect/pull/9046
+- Add `configuration` to `kill_infrastructure` signature by @desertaxle in https://github.com/PrefectHQ/prefect/pull/9250
+
+### Experimental
+- Add worker.executed-flow-run event by @bunchesofdonald in https://github.com/PrefectHQ/prefect/pull/9227
+- Emit worker lifecycle events. by @bunchesofdonald in https://github.com/PrefectHQ/prefect/pull/9249
```

Comment:
```suggestion
- Emit worker lifecycle events - [#9249] https://github.com/PrefectHQ/prefect/pull/9249
```

Reply:
Probably "Emit events for worker lifecyle"?